packages/video_player/video_player/lib/video_player.dart

```diff
@@ -714,3 +770,70 @@ class _VideoProgressIndicatorState extends State<VideoProgressIndicator> {
     }
   }
 }
+
+/// Widget for displaying closed captions on top of a video.
+///
+/// If [text] is null, this widget will not display anything.
+///
+/// If [textStyle] is supplied, it will be used to style the text in the closed
+/// caption.
+///
+/// Note: in order to have closed captions, you need to specify a
+/// [VideoPlayerController.closedCaptionFile].
+///
+/// Usage:
+///
+/// ```dart
+/// Stack(children: <Widget>[
+///   VideoPlayer(_controller),
+///   ClosedCaption(text: _controller.value.caption.text),
+/// ]),
+/// ```
+class ClosedCaption extends StatelessWidget {
```

Comment:
Is it worth writing any widget tests for this?

Reply:
Done.